src/main/java/org/embulk/util/ssl/SSLPlugins.java

```diff
@@ -172,6 +196,7 @@ public static SSLPluginConfig configure(SSLPluginTask task)
 
     public static SSLPluginConfig configure(SSLPluginTask task, DefaultVerifyMode defaultVerifyMode)
     {
+        task.getCryptoPolicy().ifPresent(policy -> Security.setProperty(CRYPTO_POLICY, policy.toString()));
```

Comment:
It looks like changing the global `java.security.Security`'s property. What happens when a plugin sets this to `LIMITED` while another plugin sets this to `UNLIMITED`?

Reply:
In this case, the last set will take effect because security policy is a global configuration.

Reply:
If so, that's not appropriate to implement in this library, right?  It means uncontrollable from each plugin's viewpoint. I'd decline this change, then.

You may want to explain what you wanted to **finally** realize by your proposal.

Reply:
Currently, In our case, a plugin(e.g. ftp) can't connect to the server because the cipher suite is restricted. We want to control this policy based on a plugin and user's configuration instead of changing a configuration of JVM.

Reply:
> In our case, a plugin(e.g. ftp) can't connect to the server because the cipher suite is restricted.

This situation is understandable, but,

> We want to control this policy based on a plugin and user's configuration instead of changing a configuration of JVM.

Why does your case need to be based on a plugin and user's configuration?  Setting it "unlimited" globally would just allow AES 256.  Isn't it sufficient for you?

If setting it globally is okay, the right place would be the Embulk core. For example, the Embulk core can configure `java.security.Security` in tbe very beginning if an Embulk System Property `crypto.policy` is set.

Anyway, as far as I saw, I was not able to find such a non-global configuration for this. Such a global configuration cannot be accepted in `embulk-util-*` librarires for Embulk plugins.

Reply:
Or, IIUC, the latest Java 8 (after 8u161) sets this `unlimited` by default.

https://www.oracle.com/java/technologies/javase/8u161-relnotes.html#JDK-8170157

Reply:
> If setting it globally is okay, the right place would be the Embulk core. For example, the Embulk core can configure java.security.Security in tbe very beginning if an Embulk System Property crypto.policy is set.
Thank you for your suggestion. After discussing we decided to set this parameter globally. I will close this PR